grammars/css.cson

```diff
@@ -561,6 +561,44 @@
         'captures':
           '1':
             'name': 'punctuation.definition.entity.css'
+          '2':
+            'name': 'entity.other.namespace-prefix.css'
+          '3':
+            'name': 'punctuation.separator.css'
+          '4':
+            'name': 'entity.other.attribute-name.css'
+          '5':
+            'name': 'keyword.operator.pattern.css'
+          '6':
+            'name': 'string.unquoted.attribute-value.css'
+          '7':
+            'name': 'string.quoted.attribute-value.css'
+          '8':
+            'name': 'punctuation.definition.string.begin.css'
+          '9':
+            'name': 'punctuation.definition.string.end.css'
+          '10':
+            'name': 'punctuation.definition.entity.css'
+        'match': '''(?x)
+          (\\[)\\s*
+          (?:(\\g<ident>|\\*)?(\\|))?     # qualified name prefix
+          ([\\w:&&[^0-9]][\\w:.-]*+)\\s*  # attribute name
+          (?:
+            ([~|^$*]?=)\\s*
+            (?:                           # attribute value
+              (\\g<ident>)
+            | ((?>([\'"])(?:[^\\\\]|\\\\.)*?(\\k<-2>)))
+            )\\s*
+          )?
+          (\\])
+          (?<ident>-?+[a-zA-Z_\\\\\\P{ASCII}-][a-zA-Z0-9_\\\\\\P{ASCII}-]*+){0}
```

Comment:
![screen shot 2016-10-09 at 7 43 52 am](https://cloud.githubusercontent.com/assets/18070858/19216568/47d89cd8-8df4-11e6-84b0-ca6477c34d08.png)

<ident-token> can contain escape sequences, which begin with a backslash. So technically, it can contain any characters (as long as you escape it) except U+0000. I don't want to include escape in the pattern of identifier coz I think it's not worth it, since authors rarely want to use escape in their ids, class names and stuff.

It's in the original pattern and I *thought* it's nice to have so I didn't remove it. Now that I think about it, I should probably remove it, since it's no good in a pattern that doesn't match whitespace.

If we were to support escape, the pattern could be something like:

```
-?(?:[_a-z]|(?![\u0000-\u0239]).*|\\[0-9a-f]{1,6}(\r\n|[ \t\r\n\f])?|\\[^\r\n\f0-9a-f])(?:[_a-z0-9-]|(?![\u0000-\u023‌​9]).*|\\[0-9a-f]{1,6‌​}(\r\n|[ \t\r\n\f])?|\\[^\r\n\f0-9a-f])*
```

Or we can simply change the pattern of identifier to `.+`, to let authors themselves decide what not to write.